### DISCUSSION TITLE
Filter by labels

### DISCUSSION THREAD

**Discussion by kotte33** [transmission/transmission#6004] [Ideas]
Current version has only 3 types of filtering - by state, by trackers, by name. I think it would be very convenient to filter by labels and take it by dropdown list, as like as filter by state or tracker.
Very close idea is #2794

**Comment by dantearmok** [comment]
Clarify please.  Once View>Use Groups is selected, the torrents are displayed within their Groups.  In Preferences, Groups can be created and re-arranged.  Are you saying you want another field(s), that acts subservient to the assigned Group?

**Reply by kotte33** [reply]
Good day! I use Transmisson on Synology NAS. And i can't find those options you talking about
![image](https://github.com/transmission/transmission/assets/20434354/dfd8ce20-2a93-4baa-ac1c-6ad74e709813)
![image](https://github.com/transmission/transmission/assets/20434354/4d77801b-af0c-42e1-98d9-cdb0cf3691a8)
Please send screenshots of this features

**Comment by dantearmok** [comment]
Ok.  Looks like the groups and such isn't implemented ?yet? for the NAS version.  It is there for the Mac version - so perhaps it's just a matter of porting it over.

@CKerr Is it possible to do this for the NAS version?

**Comment by blixten85** [comment]
I second this very much. Please, add so filter on labels can be done.
Also saved path would be very, very good. 
Preferdably a additional droplist. Right now there are two droplists. One for 'All, Active, Downloading, Seeding, Paused, Finnished' 
and the second is for the trackers. The third would be for labels/categories/saved path. 

Also, auto adding labels would be good too, from things like Sonarr and Radarr. They go into the right directory but the label does not get added automatically.  

I like this client but it is a little bit starved of navigational features in the GUI. Such as this filter on labels and saved path. 
You can't set a listening ip/network card in the Edit reference -> Networking, list either. Just the port.

**Comment by ashleyconnor** [comment]
Type `labels:<label>` in the search bar:

<img width="757" alt="image" src="https://github.com/user-attachments/assets/780680c3-b526-43ce-bec2-168d427579fd" />

[Source](https://github.com/transmission/transmission/blob/87bcf1a1d5bc60070315335a85fb8526843b28ac/web/src/transmission.js#L1030C17-L1030C24)

**Reply by kotte33** [reply]
Thanks, it works.
